File: AplBankSampah/app/src/main/java/com/example/aplbanksampah/Logout.java
package com.example.aplbanksampah;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.aplbanksampah.ui.logout.LogoutFragment;

public class Logout extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.logout_activity);
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.container, LogoutFragment.newInstance())
                    .commitNow();
        }
    }
}

File: AdminSampah/app/src/main/java/com/example/adminsampah/Logout.kt
package com.example.adminsampah

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.adminsampah.ui.main.LogoutFragment

class Logout : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.logout_activity)
        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                .replace(R.id.container, LogoutFragment.newInstance())
                .commitNow()
        }
    }
}